github/apps.go

```diff
@@ -344,6 +358,28 @@ func (s *AppsService) CreateInstallationToken(ctx context.Context, id int64, opt
 	return t, resp, nil
 }
 
+// CreateInstallationTokenListRepos creates a new installation token with an empty repository list as a parameter.
```

Comment:
I'm thinking that we should mention _why_ this method exists and what makes it different from the other one.